pallene/driver.lua

```diff
@@ -29,27 +30,21 @@ local function check_source_filename(argv0, file_name, expected_ext)
     return name
 end
 
+function driver.load_input(path)
+    local base_name, err = check_source_filename("pallenec test", path, "pln")
+    if not base_name then
+        return base_name, err
```

Comment:
We should probably choose between `if not foo then return foo` or `if not foo then return false` and use the same pattern everywhere in the file.

Reply:
I can work on that. The problem is, I am not sure if these are falsy values or an actual `false`.